tests/eas/load_tracking.py

```diff
@@ -83,14 +95,16 @@ def get_expected_util_avg(self, experiment):
         # Find the capacity of the CPU the workload was run on
         [cpu] = experiment.wload.cpus
         cpufreq = experiment.conf['cpufreq']
+
+        cpu_capacity = self._get_cpu_capacity(self.te, cpu)
         if cpufreq['governor'] == 'userspace':
             freq = cpufreq['freqs'][cpu]
+            max_freq = max(self.te.target.cpufreq.list_frequencies(cpu))
+            cpu_capacity *= float(freq) / max_freq
         else:
             assert cpufreq['governor'] == 'performance'
-            freq = None
-        cpu_capacity = self.te.nrg_model.get_cpu_capacity(cpu, freq)
 
-        scaling_factor = float(cpu_capacity) / self.te.nrg_model.capacity_scale
```

Comment:
Maybe it would be worth it to keep this line ? Admittedly 1024 is supposed to be used everywhere, but there might be some platforms with goofy cap scales ? If that's really unlikely just ignore this comment.